channeldb/graph.go

```diff
@@ -1470,7 +1478,10 @@ func deserializeChanEdgeInfo(r io.Reader) (*ChannelEdgeInfo, error) {
 			return nil, err
 		}
 
-		return btcec.ParseSignature(sigBytes, btcec.S256())
+		if len(sigBytes) != 0 {
```

Comment:
As far as I understand after discussion the `nil` signature does not designate something broken.